sailpoint-identitynow/info.json

```diff
@@ -0,0 +1,803 @@
+          "required": false,
+          "visible": true,
+          "editable": true
+        }
+      ],
+      "output_schema": [
+        {
+          "id": "",
+          "name": "",
+          "created": "",
+          "modified": "",
+          "sourceId": "",
+          "identityId": "",
+          "attributes": {},
+          "authoritative": "",
+          "description": "",
+          "disabled": "",
+          "locked": "",
+          "nativeIdentity": "",
+          "systemAccount": "",
+          "uncorrelated": "",
+          "uuid": "",
+          "manuallyCorrelated": "",
+          "hasEntitlements": ""
+        }
+      ]
+    },
+    {
+      "operation": "get_account_details",
+      "title": "Get Account Details",
+      "description": "Retrieves a account detail based on the ID.",
+      "category": "investigation",
+      "annotation": "get_account_details",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Account ID",
+          "name": "id",
+          "type": "text",
+          "tooltip": "Specify the ID of the account for which you are retrieving details",
+          "description": "Specify the ID of the account for which you are retrieving details",
+          "required": true,
+          "visible": true,
+          "editable": true
+        }
+      ],
+      "output_schema": {
+        "id": "",
+        "name": "",
+        "created": "",
+        "modified": "",
+        "sourceId": "",
+        "identityId": "",
+        "attributes": {},
+        "authoritative": "",
+        "description": "",
+        "disabled": "",
+        "locked": "",
+        "nativeIdentity": "",
+        "systemAccount": "",
+        "uncorrelated": "",
+        "uuid": "",
+        "manuallyCorrelated": "",
+        "hasEntitlements": ""
+      }
+    },
+    {
+      "operation": "get_account_activities",
+      "title": "Get Account Activities",
+      "description": "This gets a collection of account activities that satisfy the given query parameters.",
+      "category": "investigation",
+      "annotation": "get_account_activities",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Type",
+          "name": "account_type",
+          "type": "text",
+          "tooltip": "The type of account activity.",
+          "description": "The type of account activity.",
+          "required": false,
+          "visible": true,
+          "editable": true
+        },
+        {
+          "title": "Requested For",
+          "name": "requested-for",
+          "type": "text",
+          "tooltip": "The identity that the activity was requested for. me indicates the current user. Mutually exclusive with regarding-identity.",
+          "description": "The identity that the activity was requested for. me indicates the current user. Mutually exclusive with regarding-identity.",
```

Comment:
The identity that the activity was requested for. **"me"** indicates the current user. Mutually exclusive with regarding-identity.